blocks/identity-block/components/Identity.jsx

```diff
@@ -1,26 +1,41 @@
-// import { useState } from 'react';
+import { useState } from 'react';
 import Identity from '@arc-publishing/sdk-identity';
-// import { useFusionContext } from 'fusion:context';
-// import getProperties from 'fusion:properties';
-// import { isServerSide } from '@wpmedia/engine-theme-sdk';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import { isServerSide } from '@wpmedia/engine-theme-sdk';
 
-const useIdentity = () => ({
-  Identity,
-  isInitialized: true,
-});
+const useIdentity = () => {
+  const { arcSite } = useFusionContext();
+  const { subscriptions } = getProperties(arcSite);
+  const [isInit, setIsInit] = useState(() => !!Identity.apiOrigin);
+  if (!isInit && arcSite && subscriptions?.identity?.apiOrigin) {
```

Comment:
Per above: 
```suggestion
  if (!isInit && arcSite && api?.identity?.origin) {
```

Reply:
unnecessary to show optional chaining though right?

Reply:
```js  
if (!isInit && arcSite && api?.identity?.origin) {
```